online/src/main/scala/ai/chronon/online/Fetcher.scala

```diff
@@ -81,7 +80,8 @@ class Fetcher(val kvStore: KVStore,
               timeoutMillis: Long = 10000,
               logFunc: Consumer[LoggableResponse] = null,
               debug: Boolean = false,
-              val externalSourceRegistry: ExternalSourceRegistry = null)
+              val externalSourceRegistry: ExternalSourceRegistry = null,
```

Comment:
This is not backwards compatible IMO. I think we should make this nullable, and use get set methods to modify the flagStore